### ISSUE TITLE
Windows Support

### ISSUE THREAD

**Issue by kadaan** [jvanzyl/presto-maven-plugin#1]
Can you please change lines 92 & 97 to use to the correct path separator for each platform?

**Comment by jvanzyl** [comment]
Sure. I don't think the first is an issue as that's Ant scanning code which I believe does the right thing but I'll verify. The second instance looks like an issue.

**Comment by kadaan** [comment]
I agree.  I changed both to use File.separator and it works on Windows now.

**Comment by kadaan** [comment]
Thanks! I have another windows support issue that is related to takari and presto, but am having a hard time finding out where the source is and where to created the issue.  The problem is that the io.provis.provisio plugin uses takari.archiver 0.1.7.  This version has an issue with reading windows file attributes.  This issue is fixed in takari.archiver 0.1.8.  Do you know where I can file an issue to have io.provis.provisio depend on takari.archiver 0.1.8 and have a new package pushed?

**Comment by jvanzyl** [comment]
I will fix that too and push it. I will try it with Presto and if it's all good I'll push new releases and make a pull request for Presto.

**Comment by kadaan** [comment]
Thanks a ton!!!

**Comment by kadaan** [comment]
Any update on the io.provis.provisio, takari.archiver issue?  This one makes it especially difficult to build Presto on Windows because I cannot find the source and have to resource to modifying the provisio-0.1.14.pom in my maven package cache, updating takariArchiverVersion to 0.1.8.

**Comment by zygitup** [comment]
![image](https://user-images.githubusercontent.com/14799384/50804623-d59e0280-1329-11e9-88c8-878a7ddbd2f8.png)
![image](https://user-images.githubusercontent.com/14799384/50804627-db93e380-1329-11e9-826c-a289ad02eb17.png)
   presto-maven-plugin:0.1.12编译提示找不到java class，这个java类是项目中自定义的。然后使用presto-maven-plugin:0.1.12编译在windows上会报${project.build.outputDirectory}位置不可用，访问不到。但手动是可以访问的。麻烦请教一下这是什么问题？谢谢

**Comment by zygitup** [comment]
The presto-maven-plugin:0.1.12 compilation prompt cannot find the Java class, which is a custom class in the project.Then use the presto - maven plugin: 0.1.12 compile on Windows at ${project. Build. OutputDirectory} position is not available, can't access.But manual access is available.May I ask what the problem is?thank you

The wrong case:
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal io.takari.maven.plugins:presto-maven-plugin:0.1.12:generate-service-descriptor (default-generate-service-descriptor) on project carbondata-presto: Execution default-generate-service-descriptor of goal io.takari.maven.plugins:presto-maven-plugin:0.1.12:generate-service-descriptor failed: A required class was missing while executing io.takari.maven.plugins:presto-maven-plugin:0.1.12:generate-service-descriptor: org\apache\carbondata\presto\CarbonColumnVectorWrapper (wrong name: org/apache/carbondata/presto/CarbonColumnVectorWrapper)


Caused by: java.lang.NoClassDefFoundError: org\apache\carbondata\presto\CarbonColumnVectorWrapper (wrong name: org/apache/carbondata/presto/CarbonColumnVectorWrapper)

**Comment by zygitup** [comment]
> I have fixed it in my pull request. This isn't approved yet.

Are you answering my question?When will this problem be fixed?Please inform me when it is repaired. Thank you very much

**Comment by zygitup** [comment]
> I have fixed it in my pull request. This isn't approved yet.

Hello, when can I update this version? TKS

**Comment by Aqua-Dream** [comment]
> I have fixed it in my pull request. This isn't approved yet.

You pull request https://github.com/prestodb/presto-maven-plugin/pull/11 has one problem:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:2.17:check (checkstyle) on project presto-kafka: You have 43401 Checkstyle violations. -> [Help 1]
```

I suggest writing
``` java
classPath.substring(0, classPath.length() - 6).replace(File.separator, ".");
```
instead of 
``` java
classPath.substring(0, classPath.length() - 6).replace(File.separator, '.');
```
because it seems that "File.separator" will return a string instead of a char.

**Comment by Megaprog** [comment]
Hello! I do not use "File.separator" in my pull request #11 but "File.separateChar"

**Comment by Aqua-Dream** [comment]
> Hello! I do not use "File.separator" in my pull request #11 but "File.separateChar"

That's right. Sorry for this.
